sites/public/pages/account/dashboard.tsx

```diff
@@ -46,6 +46,12 @@ function Dashboard(props: DashboardProps) {
     </span>
   )
 
+  const myFavoritesIcon = (
+    <span className="header-badge">
+      <Icon size="medium" symbol="likeFill" fill="#E94761" iconClass="ui-icon__filledRed" />
```

Comment:
Curious why we are passing an icon color as a class instead of using the fill prop?

Reply:
yeah, not super stocked about doing it that way, but we apply the rule `.ui-icon svg` to all svgs which overwrites the fill, so I had to build a class that would overwrite the overwrite 

I thought about pulling that rule but I'm not sure if we should do that in this ticket

Reply:
Ohhhh yeah that global override has been annoying, we can make a separate issue for sure

Reply:
https://github.com/CityOfDetroit/bloom/issues/1102